src/Tools/Performance Toolkit/App/src/BCPTLine.Codeunit.al

```diff
@@ -346,4 +351,11 @@ codeunit 149005 "BCPT Line"
         Parm := format(FldRef.Value, 0, 9);
         exit(true);
     end;
+
+    local procedure OnBeforeAddLogEntry(var BCPTLine: Record "BCPT Line"; var Operation: Text; var ExecutionSuccess: Boolean; var Message: Text; var Handled: Boolean)
+    var
+        BCPTTestSuite: Codeunit "BCPT Test Suite";
+    begin
+        BCPTTestSuite.OnBeforeBCPTLineAddLogEntryWrapper(BCPTLine, Operation, ExecutionSuccess, Message, Handled);
```

Comment:
not your issue but what does the 'Wrapper' bit on the end mean?

Reply:
oh, it is your issue... damn... yeah, what extra info does the word wrapper convey here?

Reply:
I don't know if the wrapper is proper naming convention. It is just tool to make an event available for other apps and for the programmer who reads the code can follow the thread easily to which event he/she should really subscribe on.